client/src/modules/shopping-list/model/reducer.js

```diff
@@ -80,19 +80,19 @@ const lists = (state = {}, action) => {
         [action.payload.listId]: action.payload.data
       };
     }
-    case ItemActionTypes.ADD_ITEM_SUCCESS: {
+    case ItemActionTypes.ADD_SUCCESS: {
       return {
         ...state,
         [action.payload.listId]: items(state[action.payload.listId], action)
```

Comment:
Create const currentList = state[action.payload.listId]  before return ,
and pass this const to items reducer 
In every case